docs/001_develop/01_development-environment/003_genesis-start/index.mdx

```diff
@@ -11,57 +11,59 @@ tags:
 sidebar_position: 3
 ---
 
-Genesis Start is a small program which allows you to preview an application, including during the development cycle, by running it on your local desktop. 
+Genesis Start enables you to run an application on your local desktop. This is useful when you are developing locally.
 
-From the command line, go to your project's `server` directory, run:
+From the command line, go to your project's **server** directory and run:
 
 ```shell
 ./gradlew genesisStart
 ```
 
-If the command fails, you may need to [enable Genesis Start](#enabling-genesis-start)
+If the command fails, you probably need to [enable Genesis Start](#enabling-genesis-start)
 
-Genesis start provides a simple UI to run your app.
+Genesis Start provides a simple UI to run your app.
+
+A list of processes is displayed in the main area, and a side toolbar provides buttons for selecting different actions.
 
 ![Genesis Start Full Application](/img/genesis-start/genesis-start-full-app.png)
 
-## How to use
+## Using Genesis Start
 
-### Bootstap and launch 
+### Bootstrap and launch 
 ![Bootstrap and Launch](/img/genesis-start/genesis-start-bootstrap.png)
 
-Bootstrap the application with a single click.
+This button bootstraps the application with a single click.
 
 It is helpful when:
-- Starting an application for the first time
-- Starting an application having made changes to the database schema
+- starting an application for the first time
+- starting an application after you have made changes to the database schema
 
 :::info
-Bootstrap first runs **remap**, imports project data, then runs **ConsolidateRights**, and finally **Start All**. 
+Bootstrap first runs **remap**, imports the project data, then runs **ConsolidateRights**, and finally **Start All**. 
 :::
 
 ### Import data
 ![Import data](/img/genesis-start/genesis-start-import-data.png)
 
-Select csv files to import into your application's database. Uses the `SendIt`(/build-deploy-operate/operate/commands#sendit) command.
+This button enables you to select csv files to import into your application's database. It uses the Genesis `SendIt`(/build-deploy-operate/operate/commands#sendit) command to do this.
```

Comment:
Just realised we are missing [] around `SendIt` (I was too!)
![image](https://github.com/user-attachments/assets/2f418be1-63ef-4d2f-a7bc-e7958233f5cb)

Reply:
done